main/libjpeg-turbo/APKBUILD

```diff
@@ -1,52 +1,43 @@
+# Contributor: Johan Bergström <bugs@bergstroem.nu>
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libjpeg-turbo
-pkgver=1.5.3
-pkgrel=4
+pkgver=2.0.2
+pkgrel=0
 pkgdesc="accelerated baseline JPEG compression and decompression library"
 url="https://libjpeg-turbo.org/"
 arch="all"
 license="IJG"
 depends=""
-makedepends="nasm"
+makedepends="cmake nasm"
 replaces="libjpeg"
 subpackages="$pkgname-doc $pkgname-dev $pkgname-utils"
-source="https://downloads.sourceforge.net/libjpeg-turbo/libjpeg-turbo-$pkgver.tar.gz
-	0001-tjLoadImage-Fix-FPE-triggered-by-malformed-BMP.patch
-	CVE-2018-11813.patch
-	"
-
-# secfixes:
-#   1.5.3-r3:
-#   - CVE-2018-11813
-#   1.5.3-r2:
-#   - CVE-2018-1152
```

Comment:
Why are you removing secfixes metadata?

Reply:
I couldn't find documentation (looked [here](https://wiki.alpinelinux.org/wiki/APKBUILD_Reference) among other things) on how to cover cases which patches were no longer in the repo. These should stay permanently?